### ISSUE TITLE
Run command on click or wheel event

### ISSUE THREAD

**Issue by RaitaroH** [Zren/plasma-applet-commandoutput#6]
Would be nice if when clicked the applet it will do a customized thing. Say for example, I have my little check for updates with pkcon script that I talk about ![here](https://github.com/Zren/plasma-applet-terminalupdate/issues/2). Would be really sweet if you could have a custom command for when you click on the plasmoid such as in this case something like `konsole -e "sudo apt-get dist-upgrade -y"`

In essence this is similar to i3-blocks. I suppose if possible a middle click, double click, roll the mouse wheel would also be really cool. I know in the `event calendar` there are mouse wheel commands so why not? :)

**Comment by Zren** [comment]
Added to v6

**Comment by RaitaroH** [comment]
The click and wheel is awesome... but is volatile memory it seems. Once set is remembered until latte or plasma is killed.

To be safe I decided to reinstall everything instead of `bash reinstall`: 

```
▶ kpackagetool5 -t Plasma/Applet -r package
Empty filename passed to function
kf5.kpackage: Generated  "/home/raitaro/.local/share/plasma/plasmoids//kpluginindex.json"  ( 4  plugins)
Successfully uninstalled /home/raitaro/github/plasma-applet-commandoutput/package

▶ git clone https://github.com/Zren/plasma-applet-commandoutput.git
▶ cd plasma-applet-commandoutput

▶ bash build
[build] translate dir found, running merge.
./merge: 7: ./merge: Bad substitution
[merge] Extracting messages
[merge] Done extracting messages
[merge] Merging messages
[merge] ./nl_NL.po
......... done.
[merge] Done merging messages
./build: 8: ./build: Bad substitution
[build] Compiling messages
./nl_NL.po
[build] Install to /home/raitaro/github/plasma-applet-commandoutput/package/translate/../contents/locale/nl_NL/LC_MESSAGES/plasma_applet_com.github.zren.commandoutput.mo
[build] Done building messages
[build] (re)install the plasmoid and restart plasmashell to test.
rm: cannot remove 'commandoutput-v*.plasmoid': No such file or directory
adding: contents/ (stored 0%)
adding: contents/locale/ (stored 0%)
adding: contents/locale/nl_NL/ (stored 0%)
adding: contents/locale/nl_NL/LC_MESSAGES/ (stored 0%)
adding: contents/locale/nl_NL/LC_MESSAGES/plasma_applet_com.github.zren.commandoutput.mo (deflated 39%)
adding: contents/config/ (stored 0%)
adding: contents/config/main.xml (deflated 69%)
adding: contents/config/config.qml (deflated 26%)
adding: contents/ui/ (stored 0%)
adding: contents/ui/main.qml (deflated 65%)
adding: contents/ui/lib/ (stored 0%)
adding: contents/ui/lib/ConfigTextAlign.qml (deflated 69%)
adding: contents/ui/lib/ConfigPage.qml (deflated 48%)
adding: contents/ui/lib/ConfigSection.qml (deflated 59%)
adding: contents/ui/lib/ConfigTextFormat.qml (deflated 73%)
adding: contents/ui/lib/ConfigCheckBox.qml (deflated 48%)
adding: contents/ui/lib/ConfigFontFamily.qml (deflated 37%)
adding: contents/ui/lib/ConfigSpinBox.qml (deflated 62%)
adding: contents/ui/lib/AppletVersion.qml (deflated 52%)
adding: contents/ui/lib/ConfigComboBox.qml (deflated 64%)
adding: contents/ui/config/ (stored 0%)
adding: contents/ui/config/ConfigGeneral.qml (deflated 68%)
adding: metadata.desktop (deflated 45%)
adding: translate/ (stored 0%)
adding: translate/plasmoidlocaletest (deflated 61%)
adding: translate/install (deflated 55%)
adding: translate/nl_NL.po (deflated 62%)
adding: translate/build (deflated 53%)
adding: translate/ReadMe.md (deflated 51%)
adding: translate/merge (deflated 61%)
adding: translate/template.pot (deflated 65%)
md5: ...

▶ bash install
kf5.kpackage: Generated  "/home/raitaro/.local/share/plasma/plasmoids//kpluginindex.json"  ( 5  plugins)
Successfully installed /home/raitaro/github/plasma-applet-commandoutput/package
```

As you can see v7. But once I have latte replaced, or I have a default panel reset by restarting plasma... the custom click command is just gone. Restarting the laptop has the same effect.

![screenshot_20190215_160458](https://user-images.githubusercontent.com/19552720/52861534-765abb80-313b-11e9-994a-608620c6af27.png)

**Comment by Zren** [comment]
I can reproduce, but it does not make sense. I'm using the exact same code as "command", and it is serialized properly. I looked at `~/.config/plasma-org.kde.plasma.desktop-appletsrc` and confirmed that that while `command`, `bold`, and `fontSize` are saved to file, `clickCommand`, `mousewheelUpCommand`, and `mousewheelDownCommand` are not. 

This is really weird, as the only way the config window can communicate with the widget on the desktop is to save the property to the config.

**Comment by Zren** [comment]
It appears that KDE does not like my commented out

```xml
<!-- <default>kdialog --yesno "test"</default> -->
```

I noticed that SublimeText was highlighting the `--` in `--yesno` as an error, but I thought that was a bug in the SublimeText syntax. Turns out it's probably an xml keyword or something. Notice how xml comments end with `-->`? I'll either delete the line with the commented out preset for testing or figure out the proper way to escape it.

`config/main.xml`
```xml
		<entry name="clickCommand" type="string">
			<default></default>
			<!-- <default>kdialog --yesno "test"</default> -->
		</entry>
```
